R/utils.R

```diff
@@ -167,3 +167,19 @@ is_outdated <- function(src, dst) {
 }
 
 is_testing = function() identical(Sys.getenv("TESTTHAT"), "true")
+
+local_test_package <- function(..., .envir = parent.frame()) {
```

Comment:
love the look of this, and looks like a great redesign of generating test packages (the current suite is pretty unwieldy/arbitrarily organized). a good companion would be `local_translation_conn()` to keep translations right next to where they'll be used too.

but feels like it should be in `tests/testthat/helper.R`, no?